index.html

```diff
@@ -68,5 +68,166 @@ <h3 class="title">I'am a Software Developer</h3>
              </li>
         </ul>
     </section>
+    <section class="work">
+      <h2 class="work-header">My recent Works</h2>
+      <div class="works flex">
+        <img class="img-works" src="./images/main-project-setup.jpg" alt="project" />
```

Comment:
I suggest you use background images here as stated in the project requirements: 
(decorations are always background images, not `img` tags)

Reply:
Hi @Davidosky007, Yes, you are right! 👍 I actually planned to do this but I must have missed it as I did in other places, see the permalinks below:

>https://github.com/bimbolabuari/mv-portfolio/blob/eed2622751568e6f614e89b29e8203ca3ddb97f5/styles.css#L50

Reply:
Nice 👍